runtime/tools/python/setup.py

```diff
@@ -62,6 +63,24 @@
             f"{metallibdir}/{dylib}",
             f"{src_dir}/build/runtime/tools/python/ttrt/runtime",
         )
+        command = [
+            "patchelf",
+            "--set-rpath",
+            "$ORIGIN",
+            f"{src_dir}/build/runtime/tools/python/ttrt/runtime/{dylib}",
+        ]
```

Comment:
Can you test on macOS?  I don't think `patchelf` exists there, macOS technically has install_name_tool, but let's not bother because I don't think we need standalone build support for macOS.  Hopefully if we can get the rpath cmake thing figured out macOS would just work.

Reply:
For now, let's only run `patchelf` on linux.

Reply:
Mac has patchelf installation via brew. I tried just patching the binary itself works. But since this is under `if enable_runtime:` shouldn't it only be run on Ubuntu 22 anyways? By default, users shouldn't be building ttrt on Mac for runtime.